overwolf/src/logic/friends.ts

```diff
@@ -0,0 +1,25 @@
+const serverUrl = process.env.SERVER_URL || 'http://localhost:8000/data/update';
+
+export async function updateFriendLocation(id: string, name: string, location: Vector2, friends: string) {
```

Comment:
Ey! Nice work! 👏 

Just curious, have you choose an REST schema for any special reason? When I saw the issue I thought that a socket system may improve connection, server performance, scalability and user experience.

Reply:
Initially I just made the server for testing purpose, I already did something like that so it was fast to do. I do not have any preference, a socket connection is okay too :)

Reply:
I think REST is _good enough_ (at least for now). Sure web sockets might improve the response times, but since we would probably only get an update every second, a simple GET request shouldn't be much of a problem and would simplify the logic of both server and app tremendously.